test/hotspot/jtreg/runtime/ErrorHandling/libTestDwarfHelper.h

```diff
@@ -21,9 +21,11 @@
  * questions.
  */
 
-#include "jni.h"
 #include <stdio.h>
```

Comment:
Seems unneeded.

Reply:
So what I did here was change:
```
#include "jni.h"
#include <stdio.h>
```

into 
```
#include <stdio.h>

#include "export.h"
#include "jni.h"
```

The reordering was strictly not needed, but putting user includes in front of system ones looked like bad coding to me, and put me in a difficult spot in where to add the new `#include "export.h"` -- next to the current user include even if I thought that was wrong, or have the system includes "sandwitched" between two user includes.

Or do you mean that it seems unneeded to include `jni.h` at all? Yes, I agree, but it was there before, and I don't want to make any other changes to the tests. This change is scary enough as it is :-). If you want, I can file a follow-up to remove this include instead.